src/components/Select/Select.tsx

```diff
@@ -108,7 +108,10 @@ const Select: React.FunctionComponent<SelectProps> = ({ options, onChange }) =>
   const [selectedOptionIndex, setSelectedOptionIndex] = useState(0)
   const [containerSize, setContainerSize] = useState({ width: 0, height: 0 })
 
-  const toggling = () => setIsOpen(!isOpen)
+  const toggling = (event: React.MouseEvent<HTMLDivElement>) => {
+    setIsOpen(!isOpen)
+    event.stopPropagation()
```

Comment:
Why `stopPropagation`?

Reply:
It propogates through to document that we also listen click event which causes close the dropdown.